poc/gen_test_vec.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Don't we also need to change the file permissions to make this executable?

Reply:
I did so, this is indicated as "100644 → 100755" next to the filename in GitHub's UI.